viminfo_test.go

```diff
@@ -20,16 +20,24 @@
 // IN THE SOFTWARE.
 //
 
-package viminfo
+package viminfo_test
```

Comment:
I'm in the other camp. Only use a separate test package when its absolutely warranted -- otherwise, test from the default package.

Reply:
I do prefer testing "from outside" better, but it's not a big thing.